### PR TITLE
Changes from trac 164 discussion

### PR DESCRIPTION
**Pull request by twhiteaker**
See http://cf-trac.llnl.gov/trac/ticket/164

 - [ ] Added link from trac ticket to this PR?

 > Applied via ​https://github.com/cf-convention/cf-conventions/pull/115

 - [ ] Updated "Revision History"? (Use the date you applied the changes.)

### CODE REVIEW

File: appa.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 All CF attributes are listed here except for those that are used to describe grid mappings. See Appendix F for the grid mapping attributes.
 
-The "Type" values are **S** for string, **N** for numeric, and **D** for the type of the data variable. The "Use" values are **G** for global, **C** for variables containing coordinate data, **D** for variables containing non-coordinate data, and **-** for variables with a special purpose. "Links" indicates the location of the attribute"s original definition (first link) and sections where the attribute is discussed in this document (additional links as necessary).
+The "Type" values are **S** for string, **N** for numeric, and **D** for the type of the data variable. The "Use" values are **G** for global, **C** for variables containing coordinate data, **D** for variables containing non-coordinate data, **M** for geometry container variables, and **-** for variables with a special purpose. "Links" indicates the location of the attribute"s original definition (first link) and sections where the attribute is discussed in this document (additional links as necessary).
```
Comment (davidhassell) [inline appa.adoc]: I've nothing against it - but why **M**?

Comment (dblodgett-usgs) [reply appa.adoc]: I would guess it stands for "Mapping" ? Should we change it to "G" for geometry? Not a big deal either way right?

Comment (twhiteaker) [reply appa.adoc]: G was already taken for "global".  I also wanted C for "container" but that was taken for "coordinate data".  Then I thought "V" for geometry container Variable but that seemed too generic. So I went with M for geoMetry.  I'm not enamored with it and am totally open to suggestions.  If we can use two characters, we could do GC.

Comment (dblodgett-usgs) [reply appa.adoc]: Fine by me!

**Pull request merged**